### PR TITLE
BENCH-132-dependabot

### PR DESCRIPTION
**Pull request by PietroConvalleAD**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "pip" # See documentation for possible values
```
Comment (JoeCSykes) [inline .github/dependabot.yml:8]: It think our package-ecosystem is poetry no?
https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file

Comment (PietroConvalleAD) [reply .github/dependabot.yml:8]: yes, but the yaml value is "pip" https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem

**Review by JoeCSykes — APPROVED** [review]

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "weekly"
```
Comment (abbas-khan8) [inline .github/dependabot.yml:11]: What made you choose weekly?

**Review by abbas-khan8 — APPROVED** [review]

File: .github/dependabot.yml (hunk 2 above)
Comment (PietroConvalleAD) [reply .github/dependabot.yml:11]: default value, I can change it I guess

Comment (abbas-khan8) [reply .github/dependabot.yml:11]: No need to, just wondering! Weekly should be fine :)

**Pull request merged**